src/shared/ui/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component, ReactNode } from 'react';
+import css from './index.module.css';
+
+interface Props {
+  children: ReactNode;
+}
+
+interface State {
+  hasError: boolean;
+  error: Error | null;
+}
+
+export class ErrorBoundary extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { hasError: false, error: null };
+  }
+
+  static getDerivedStateFromError(error: Error) {
+    return { hasError: true, error };
+  }
+
+  reloadPage = () => {
+    window.location.reload();
+  };
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <div className={css.body}>
+          <h2>Oops..! Looks like something gone wrong... 🐈 (reload page!)</h2>
+
```

Comment:
think it was a little bit separate title from button, actually idk we need this line or not
so, we can remove, because it's not necessary (and no hard logical here which we should separate🤔)

i remove this line